gui/frontend/tests/pages/Networks.test.tsx

```diff
@@ -0,0 +1,27 @@
+import { render, screen } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import { describe } from 'vitest';
+
+import Networks from '../../src/pages/Networks';
+import { NetworksContextProvider } from '../../src/store/NetworkContext';
+
+describe('App', () => {
+  it('renders title', () => {
+    (window as any)['go'] = {};
```

Comment:
will look for a cleaner way to handle this mocking. wails docs dont have a section on testing